metrics/record_test.go

```diff
@@ -186,3 +188,37 @@ func TestBucketsNBy10(t *testing.T) {
 		})
 	}
 }
+
+func TestMeter(t *testing.T) {
+	measure := stats.Int64("request_count", "Number of reconcile operations", stats.UnitNone)
+	// Increase the measurement value for each test case so that checking
+	// the last value ensures the measurement has been recorded.
+	meterTestCases := []cases{{
+		name:          "resource 1",
+		metricsConfig: &metricsConfig{},
+		measurement:   measure.M(1),
+		resource:      resource.Resource{Type: "resource 1", Labels: map[string]string{"bar": "foo"}},
+	}, {
+		name:          "resource 2",
+		metricsConfig: &metricsConfig{},
+		measurement:   measure.M(2),
+		resource:      resource.Resource{Type: "resource 2", Labels: map[string]string{"bar": "foo", "bar1": "foo1"}},
+	}}
+	v := &view.View{
+		Measure:     measure,
+		Aggregation: view.LastValue(),
+	}
+	RegisterResourceView(v)
+	defer UnregisterResourceView(v)
```

Comment:
t.Cleanup? We're moving to that most of places.

Reply:
Done.